bedrock/products/templates/products/vpn/resource-center/includes/vpn-article-card.html

```diff
@@ -0,0 +1,16 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% from 'macros-protocol.html' import card %}
+
+{{ card (
+  title=article.info.title,
+  ga_title=article.info.title,
+  aspect_ratio='mzp-has-aspect-16-9',
+  desc=article.info.blurb,
+  image_url=article.info.seo.image or "img/products/vpn/resource-center/article-1-light.svg",
```

Comment:
should we loop through the images? article-1-light, article-2-light, etc? I'm not sure where those variations are used.

Reply:
I think i can just remove the images we're not using since all other images will come from contentful